src/AppBridgeBlock.spec.ts

```diff
@@ -3,15 +3,15 @@
 import { Mock, afterEach, beforeAll, describe, expect, it, test, vi } from 'vitest';
 import { AppBridgeBlock } from './AppBridgeBlock';
 import { TerrificEvent } from './types';
-import { FrontifyColorPaletteDummy } from './tests/FrontifyColorPaletteDummy';
+import { FrontifyColorPaletteCreateDummy, FrontifyColorPaletteDummy } from './tests/FrontifyColorPaletteDummy';
```

Comment:
again, there are a few others as well...
```suggestion
import { FrontifyColorPaletteCreateDummy, FrontifyColorPaletteDummy } from './tests';
```